postownik/app/client/src/App.svelte

```diff
@@ -1,9 +1,8 @@
 <script>
-  import List from './Panels/List.svelte';
-  import { authenticated } from './store.js';
-  $: console.log($authenticated);
```

Comment:
Obaj edytowaliście App.js xD będzie konflikt. A ta autentykacja będzie musiała się odbywać w głównym pliku więc nie ma sensu tego wyrzucać. Chociaż tutaj oczywiście wrzuciłem to dla przykładu żeby dać hint gdzie to się powinno dziać, ale będzie to wyglądać trochę poważniej xD